package.json

```diff
@@ -93,23 +75,15 @@
     "bundler": "webpack --mode=production",
     "bundle:test": "webpack-dev-server --host 0.0.0.0",
     "postinstall": "node packageScripts/postinstall.js",
-    "cssLint": "stylelint \"./src/*.css\"",
     "dev": "npm run sassBuild:watch",
     "distJS": "copyfiles -u 1 -V './src/**/*.js' ./dist",
-    "sassRender": "sass-render src/*.css -t ./scripts/staticStyles-template.js",
     "sweep": "rm -rf ./temp ./demo/css ./demo/fonts ./dist ./src/tokens | rm ./src/*.css ./src/*-css.js",
     "esLint": "./node_modules/.bin/eslint src/auro-*.js",
-    "lint": "npm-run-all cssLint esLint",
+    "lint": "npm-run-all esLint",
```

Comment:
Was this intentional? And if so, can it just be `npm run esLint`?

Reply:
Yeah, all the CSS processing was removed from this component. It just returns a string and assumes the parent styling. But yeah, I see what you are saying there.